xbmc/music/windows/GUIWindowMusicNav.cpp

```diff
@@ -549,14 +549,14 @@ void CGUIWindowMusicNav::GetContextButtons(int itemNumber, CContextButtons &butt
         // 4. specific per album
         buttons.Add(CONTEXT_BUTTON_SET_CONTENT, 20195);
       }
-      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtist().size() > 0)
+      if (item->HasMusicInfoTag() && item->GetMusicInfoTag()->GetArtistString().size() > 0)
```

Comment:
Might be better to use ```!item->GetMusicInfoTag()->GetArtistString().empty()``` here.
Should improve readability a bit. Same for below.